seatunnel-engine/seatunnel-engine-common/src/main/resources/seatunnel.yaml

```diff
@@ -20,6 +20,7 @@ seatunnel:
         backup-count: 1
         queue-type: blockingqueue
         print-execution-info-interval: 60
+        print-job-metrics-info-interval: 60
```

Comment:
Default value of `print-job-metrics-info-interval` is 60. This is valid without change

Reply:
I know that the default value is also used if it is not changed here, but here is a parameter to let you know where you can change it.
Otherwise, if you run an example in the project and want to change the parameters, the results will not work

Reply:
> Default value of `print-job-metrics-info-interval` is 60. This is valid without change

I think this parameter can be added here, at least adding the default value will not cause any other bad effects
But it worked for me, at first I wanted to change the configuration, I didn't know where to change, I searched a lot of places in the project, and it didn't take effect after the change, and then it was someone else who pointed out that I knew to change here

Reply:
get

Reply:
You can configure it in your local environment. But do not commit it. Please revert it.